pages/assessment.tsx

```diff
@@ -54,28 +54,35 @@ const Assessment: NextPage = () => {
 
   const startIndexBatch = currentPage * maxItemsPerPage - maxItemsPerPage;
   const endIndexBatch = currentPage * maxItemsPerPage;
-  
-   // to make sure setAssessments is being set, otherwise it is empty
-   useEffect(() => {
+
+  // to make sure setAssessments is being set, otherwise it is empty
+  useEffect(() => {
     if (assessments.length == 0) {
       setAssessments(p);
     }
   }, [assessments.length, p]);
 
   const changePage = (direction: string): void => {
-      if (direction == "back") {
+    if (direction == "back") {
       setCurrentPage(currentPage - 1);
     } else if (direction == "next") {
       appendReAssessments(assessments.slice(startIndexBatch, endIndexBatch));
       setCurrentPage(currentPage + 1);
+      saveBatch(assessments.slice(startIndexBatch, endIndexBatch), taskNumber);
```

Comment:
I was just wondering if there is a reason for using the `saveBatch` after setting changing current page? 
I dont think it has gotten to actually change, but I dont know if this is something that will always will happen or sometimes glitch?

Reply:
Nope, I just put it there because it is amongst the last things to do. I can move it so it is saved before changing current page to avoid any mistakes on the slicing of the batch.